haystack/utils/docker.py

```diff
@@ -1,5 +1,6 @@
 import logging
 from typing import List, Union, Optional
+from haystack.nodes._json_schema import load_schema
```

Comment:
do you think we should add an import of `load_schema` to  `__init__.py` inside `haystack.nodes` and import from `haystack.nodes` directly? 

`_json_schema` looks ugly to me. But I am also okay to keep it here because it's a utils file.

Reply:
I don't like it either, but I wanted to keep the changes in this PR at the minimum. I also agree being an utility module makes it less problematic